bfv/evaluator.go

```diff
@@ -160,14 +167,63 @@ func (eval *evaluator) getElemAndCheckUnary(op0, opOut Operand, opOutMinDegree u
 	return // TODO: more checks on elements
 }
 
+func (eval *evaluator) TtoQ(p *Plaintext, pol *ring.Poly) {
```

Comment:
Seems like a duplicated implementation here. See my other comment.

Reply:
made tToQ private and generale